### PR TITLE
pick up .service file from ./rpm

### PR DESCRIPTION
**Pull request by kmansoft**

### CODE REVIEW

**Comment by patsevanton** [comment]
After merge i propose create think about envinment config, run as user, add selinux

**Comment by patsevanton** [comment]
@kmansoft поменяй параметры
```
Restart=on-failure
RestartSec=1
```

File: package/victoria-metrics.service [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 
 [Service]
 Type=simple
 StartLimitBurst=5
 StartLimitInterval=0
-Restart=on-failure
-RestartSec=1
-ExecStart=/usr/sbin/victoria-metrics -storageDataPath /var/lib/victoria-metrics-data -retentionPeriod 12
+Restart=on-abort
+ExecStart=/usr/local/bin/victoriametrics -storageDataPath=/var/lib/victoria-metrics-data start
```
Comment (valyala) [inline package/victoria-metrics.service]: @patsevanton , what does `start` mean in the end of this line?

**Comment by kmansoft** [comment]
And do we need `-retentionPeriod 12` or not?

File: package/victoria-metrics.service (hunk 1 above)
Comment (patsevanton) [reply package/victoria-metrics.service]: @valyala changed to 
```
ExecStart=/usr/local/bin/victoriametrics -storageDataPath=/var/lib/victoria-metrics-data
```

**Comment by patsevanton** [comment]
> And do we need `-retentionPeriod 12` or not?

May be move to EnvironmentFile

Created. Need check.
https://github.com/patsevanton/victoriametrics-rpm/tree/master/propose

**Comment by kmansoft** [comment]
```
Restart=on-failure
RestartSec=1
```

Updated

**Comment by patsevanton** [comment]
@kmansoft 
```
ExecStart=/usr/local/bin/victoriametrics -storageDataPath=/var/lib/victoria-metrics-data
```

File: package/victoria-metrics.service (hunk 1 above)
Comment (valyala) [reply package/victoria-metrics.service]: @kmansoft , then remove the trailing `start` from here

**Comment by kmansoft** [comment]
> @kmansoft
> 
> ```
> ExecStart=/usr/local/bin/victoriametrics -storageDataPath=/var/lib/victoria-metrics-data
> ```

Done

**Comment by valyala** [comment]
Thanks!

**Pull request merged**